File: Data Structures & Algorithms/majority-element/submission-1.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        map = {}
        for num in nums:
            map[num] = map.get(num, 0) + 1
        for key in map:
            if map[key] >= len(nums) // 2:
                return key

File: Data Structures & Algorithms/sort-colors/submission-1.java
class Solution {
    public void sortColors(int[] nums) {
        int[] bucket = new int[3];
        for(int num : nums){
            bucket[num]++;
        }
        int index = 0;
        for(int i = 0;i < 3;i++){
            while(bucket[i] != 0){
                nums[index] = i;
                bucket[i]--;
                index++;
            }
        }
        
    }
}